packages/shared/src/components/utilities.tsx

```diff
@@ -70,10 +71,22 @@ const themeColors = {
     button: 'btn-primary',
   },
 };
+
 export const getThemeColor = (color: string, fallback: string): ThemeColor => {
   return themeColors[color] ?? themeColors[fallback];
 };
 
+const themeFonts = {
+  bodyBold: 'font-bold typo-body',
+  body: 'typo-body',
+  title: 'typo-title3',
```

Comment:
I suggest we either add the number itself or the number word so the property will be accurate because we have an actual class for `typo-title`. Kindly coordinate with the product team so they can be wary of the values they actually use.
```suggestion
  title3: 'typo-title3',
```